airbyte-integrations/connectors/source-iterable/unit_tests/test_source.py

```diff
@@ -2,10 +2,35 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+from unittest.mock import patch
```

Comment:
`mocker.patch` can patch objects if needed, so you don't need to import `patch` and can use the `mocker` fixture in your fixtures and call `mocker.patch`

Reply:
mocker requires pytest-mock to be installed. So I just use what was in initial source-iterable code

Reply:
in order to use mocker we need to install additional lib.